### PR TITLE
Feature/forms for posts and comments added

### PR DESCRIPTION
**Pull request by Metaverse-Nabeel**
### The following changes have been made to this PR:

**Following is the list of the forms created:**

- [ ] Form for creating Comment
- [ ] Form for creating Post
- [ ] Managed all tasks using rails `ApplicationController`
- [ ] Fixed Linter errors
- [ ] Updated Readme file

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <body>
   <%= render 'single_post' %>
     <%= render_comments_section(@post) %>
+    <%= render 'shared/like' %>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/show.html.erb:8]: - [x] when i click on the like button, like count does not increment, kindly fix this

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Metaverse-Nabeel  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (Metaverse-Nabeel) [reply app/views/posts/show.html.erb:8]: Hello @VICTOR-CHUKWU 

Thank you for your feedback. I have made the correction. Now, it works as per requirement.

Regards

Comment (peterrobert) [reply app/views/posts/show.html.erb:8]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Metaverse-Nabeel, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**